.github/workflows/linter.yml

```diff
@@ -7,7 +7,22 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
+      with:
+        ref: ${{ github.ref }}
+    - name: Cache Gems
+      uses: actions/cache@v1
+      with:
+        path: vendor/bundle
+        key: ${{ runner.os }}-gem-${{ hashFiles('**/Gemfile.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-gem-
+    - name: Setup ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby }}
+    - run: |
+        gem install bundler
+        bundle config path vendor/bundle
+        bundle install --jobs 4 --retry 3
     - name: Rubocop checks
-      uses: gimenete/rubocop-action@1.0
```

Comment:
This prebuilt action only installed the latest version of `rubocop`, and not the one contained in the gemspec. While it is ideal to pull the latest version, we are at a much older version. Due to this, there are a significant number of changes to be made, and in general, it's ideal to have more control over which version is used.

The changes above are copied from the Ruby action below, and facilitate running the RuboCop check in a custom way, using the `rubocop` version specified in the gemspec.